build.sbt

```diff
@@ -401,7 +420,7 @@ lazy val bench = project.dependsOn(macrosJVM, coreJVM, freeJVM, lawsJVM)
   .settings(commonJvmSettings)
   .settings(coverageEnabled := false)
   .settings(libraryDependencies ++= Seq(
-    "org.scalaz" %% "scalaz-core" % "7.2.15"))
+    "org.scalaz" %% "scalaz-core" % "7.2.19"))
```

Comment:
`7.2.20` could be used here. Very minor though

Reply:
Yep, small point but the small points tend to add up over time. Besides, might as well add this whilst doing other changes - thx for catching this :+1:

Reply:
Thank you 👍